File: bank-extract-api/src/main/java/br/com/bank/bank/bankextractapi/adapter/in/web/dto/ActivityDto.java
package br.com.bank.bank.bankextractapi.adapter.in.web.dto;

import lombok.Data;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
public class ActivityDto {

    private Long id;
    private String sourceAccount;
    private String targetAccount;
    private String type;
    private BigDecimal value;
    private BigDecimal balance;
    private LocalDateTime date;

}

File: bank-api/src/main/kotlin/br/com/bank/adapter/in/web/dto/ActivityDto.kt
package br.com.bank.adapter.`in`.web.dto

import java.math.BigDecimal
import java.time.LocalDateTime

data class ActivityDto(val id: Long,
                       val sourceAccount: String?,
                       val targetAccount: String?,
                       val type: String,
                       val value: BigDecimal,
                       val balance: BigDecimal,
                       val date: LocalDateTime)